### PR TITLE
DOC-3363 Fix broken links in Corda OS 4.6 pages

### PR DESCRIPTION
**Pull request by michelemarsden**
Covers sections (header pages and child pages):
- Corda open source 4.6 (Welcome to Corda) page
- Release notes
- Upgrading CorDapps and nodes
- Corda API

### CODE REVIEW

File: content/en/platform/corda/4.6/open-source/release-notes.md [hunk 1]
```diff
@@ -319,7 +319,7 @@ States and apps valid in Corda 3.0 and above are usable in Corda 4.5.
 
 #### Improved `killFlow` operations
 
-We have improved the existing [killFlow RPC operation](https://api.corda.net/api/corda-os/4.5/html/api/kotlin/corda/net.corda.core.messaging/-corda-r-p-c-ops/kill-flow.html), which allows node operators to terminate flows manually - in several ways:
+We have improved the existing [killFlow RPC operation](https://api.corda.net/api/corda-os/4.5/html/api/kotlin/corda/net.corda.core.messaging/-corda-r-p-c-ops/kill-flow.md), which allows node operators to terminate flows manually - in several ways:
```
Comment (taylorharvey-r3) [inline content/en/platform/corda/4.6/open-source/release-notes.md:322]: This one just redirects to the API site, but should link to this specific function.
```suggestion
We have improved the existing [killFlow RPC operation](../../../../../en/api-ref/corda/4.6/open-source/kotlin/corda/net.corda.core.messaging/-corda-r-p-c-ops/kill-flow.html), which allows node operators to terminate flows manually - in several ways:
```

File: content/en/platform/corda/4.6/open-source/release-notes.md [hunk 2]
```diff
@@ -822,7 +822,7 @@ of tools Corda offers. Now, following the release of Corda Enterprise 4.0, we ar
 and documentation updates to bring additional stability and quality of life improvements to those developing on the Corda platform.
 
 Information on Corda Enterprise 4.0 can be found [here](https://www.r3.com/wp-content/uploads/2019/05/CordaEnterprise4_Enhancements_FS.pdf) and
-[here](https://docs.corda.net/docs/corda-enterprise/4.0/release-notes-enterprise.html). (It’s worth noting that normally this document would have started with a comment
+[here](https://docs.corda.net/docs/corda-enterprise/4.0/release-notes-enterprise.md). (It’s worth noting that normally this document would have started with a comment
 about whether or not you’d been recently domiciled under some solidified mineral material regarding the release of Corda Enterprise 4.0. Alas, we made
 that joke when we shipped the first release of Corda after Enterprise 3.0 shipped, so the thunder has been stolen and repeating ourselves would be terribly gauche.)
```
Comment (taylorharvey-r3) [inline content/en/platform/corda/4.6/open-source/release-notes.md:828]: On line 829 there's a broken link to the CE 4.0 release notes. Looks like a hard link to the old site.

File: content/en/platform/corda/4.6/open-source/app-upgrade-notes.md [hunk 3]
```diff
@@ -57,15 +57,15 @@ If you are using them you should re-namespace them to a package namespace you co
...
+3. Add Liquibase resources to CorDapps. In Corda 4.6, CorDapps that introduce custom schema need Liquibase migration scripts allowing them to create the schema upfront. For existing CorDapps that do not have migration scripts in their resources, they can be added as an external migration `.jar` file, as documented in the [Corda Enterprise documentation](../../4.6/enterprise/cordapps/database-management.html#adding-scripts-retrospectively-to-an-existing-cordapp).
 4. Update the changelog for existing schemas. After upgrading the Corda `.jar` file and adding Liquibase scripts to the CorDapp(s), any custom schemas from the apps are present
 in the database, but the changelog entries in the Liquibase changelog table are missing (as they have been created by Liquibase). This will cause issues when starting the node, and also when running `run-migration-scripts` as tables that already exist cannot be recreated. By running the new sub-command `sync-app-schemas`, changelog entries are created for all existing mapped schemas from CorDapps: `java -jar corda.jar sync-app-schemas`.
```
Comment (taylorharvey-r3) [inline content/en/platform/corda/4.6/open-source/app-upgrade-notes.md:71]: On line 137 there's an API reference that just redirects to the home page, should point here: https://docs.r3.com/en/api-ref/corda/4.6/open-source/kotlin/corda/net.corda.core.flows/-destination.html

File: content/en/platform/corda/4.6/open-source/api-flows.md [hunk 4]
```diff
@@ -1831,7 +1831,7 @@ another flow to take control and run.
 Implementations of `FlowExternalAsyncOperation` must return a `CompletableFuture`. How this future is created is up to the developer.
 It is recommended to use `CompletableFuture.supplyAsync` and supply an executor to run the future on. Other libraries can be used to
 generate futures, as long as a `CompletableFuture` is returned out of `FlowExternalAsyncOperation`. An example of creating a future
-using [Guava’s ListenableFuture](#api-flows-guava-future-conversion) is given in a following section.
+using [Guava’s ListenableFuture](#creating-completablefutures-from-guavas-listenablefutures) is given in a following section.
```
Comment (taylorharvey-r3) [inline content/en/platform/corda/4.6/open-source/api-flows.md:1835]: There is a broken link in this file here: 
image

Comment (taylorharvey-r3) [reply content/en/platform/corda/4.6/open-source/api-flows.md:1835]: Looks like an .md that needs to be an .html

Comment (taylorharvey-r3) [reply content/en/platform/corda/4.6/open-source/api-flows.md:1835]: Another here: 
image

Comment (michelemarsden) [reply content/en/platform/corda/4.6/open-source/api-flows.md:1835]: Good catch, thanks!

**Pull request merged**